client/cli/src/runner.rs

```diff
@@ -225,7 +262,7 @@ impl<C: SubstrateCli> Runner<C> {
 	where
 		FUT: Future<Output = Result<()>>,
 	{
-		run_until_exit(self.tokio_runtime, runner(self.config))
+		run_until_exit(self.tokio_runtime, runner(self.config), None)
```

Comment:
I think we should assume that the task_manager will be given in argument.

To explain to you the purpose of this function: this is a function used when someone uses sc-cli as a library and want to add new subcommands to their cli. I don't think there is any case where they won't have a task_manager to deal with.

Reply:
I'm wrong

Reply:
This is fine